robot/impl/resource_manager.go

```diff
@@ -788,7 +801,13 @@ func (manager *resourceManager) processResource(
 	resName := conf.ResourceName()
 	deps, err := r.getDependencies(ctx, resName, gNode)
 	if err != nil {
-		return nil, false, multierr.Combine(err, manager.closeResource(ctx, currentRes))
+		manager.logger.CDebugw(ctx,
```

Comment:
Why not error log?

Reply:
the error will get logged in completeConfig, this log just gives a bit more context